Src/Json/Shared.Json/ObjectDiffPatch.cs

```diff
@@ -0,0 +1,378 @@
+﻿#region *   License     *
+/*
+    SimpleHelpers - ObjectDiffPatch   
```

Comment:
Well, that NuGet package is a source-only package, so it makes sense to directly use the package. However, that code got me thinking. Wouldn't it be more useful to use the structural equivalency API (`ShouldBeEquivalentTo`) to compare the properties and fields of two `JToken` objects. I suppose those tokens contain hierarchical dictionaries that FA can inspect and compare. Thoughts?